matplotlibrc

```diff
@@ -0,0 +1,2 @@
+backend: Qt5Agg
```

Comment:
why do we need a matplotlibrc in here? will it get used when I call the executables from somewhere else?

Reply:
Also. why qt5agg? and serif?

Reply:
I accidentally added the file